docs/_index.md

```diff
@@ -43,9 +43,14 @@ curl -sSL https://install.python-poetry.org | python3 -
 
 **windows powershell install instructions**
 ```powershell
-(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | python -
+(Invoke-WebRequest -Uri https://install.python-poetry.org -UseBasicParsing).Content | py -
```

Comment:
Should it be the other way around? python by default and note to suggest use of py?

Also does Windows have an equivalent of `py || python`? Not even sure if that would work.

Reply:
Just saw your comment on the issue.

Reply:
`py` is default runner when installing from `python.org`.